### DISCUSSION TITLE
Possible Mistake in Calculating Inverse Dynamics for Floating Base in pybullet

### DISCUSSION THREAD

**Discussion by cryscan** [bulletphysics/bullet3#3380] [Q&A]
In [here](https://github.com/bulletphysics/bullet3/blob/abea1a848411cf53385fb8288c89db05e5751ef7/examples/SharedMemory/PhysicsServerCommandProcessor.cpp#L11190), when calculating inverse dynamics for floating base robots, it seems that the rest of `q` is not populated.

**Comment by rohit-kumar-j** [comment]
@cryscan are these the `q_indices` that we get from `p.getJointInfo()`?
